lib/compress/zstd_compress_sequences.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016-present, Yann Collet, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under both the BSD-style license (found in the
+ * LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ * in the COPYING file in the root directory of this source tree).
+ * You may select, at your option, one of the above-listed licenses.
+ */
+
+#ifndef ZSTD_COMPRESS_SEQUENCES_H
+#define ZSTD_COMPRESS_SEQUENCES_H
+
+// #define FSE_STATIC_LINKING_ONLY   /* FSE_encodeSymbol */ TODO is this necessary?
```

Comment:
`FSE_STATIC_LINKING_ONLY` might be necessary inside the `*.c` implementation, rather than the `*.h` declaration.

In general, we try to reduce the nb of dependencies to a minimum.
This is especially important in header files, so we try to justify each dependency with a code comment.